SimCalorimetry/HcalSimAlgos/src/HcalSiPMShape.cc

```diff
@@ -35,15 +36,28 @@ void HcalSiPMShape::computeShape() {
   // std::cout << std::endl;
 }
 
+inline double gexp(double t, double A, double c, double t0, double s) {
+  static double const root2(sqrt(2));
+  return -A*0.5*exp(c*t+0.5*c*c*s*s-c*s)*(erf(-0.5*root2/s*(t-t0+c*s*s))-1);
+}
+
 inline double onePulse(double t, double A, double sigma, double theta, double m) {
   return (t<theta) ? 0 : A*TMath::LogNormal(t,sigma,theta,m);
 }
 
-double HcalSiPMShape::analyticPulseShape(double t) const {
-  // taken from fit to laser measurement taken by Iouri M. in Spring 2016.
-  double A1(5.204/6.94419), sigma1_shape(0.5387), theta1_loc(-0.3976), m1_scale(4.428);
-  double A2(1.855/6.94419), sigma2_shape(0.8132), theta2_loc(7.025),   m2_scale(12.29);
-  return
-    onePulse(t,A1,sigma1_shape,theta1_loc,m1_scale) +
-    onePulse(t,A2,sigma2_shape,theta2_loc,m2_scale);
+double HcalSiPMShape::analyticPulseShape(double t, unsigned int signalShape) const {
+  if(signalShape==HcalShapes::ZECOTEK || signalShape==HcalShapes::HAMAMATSU){
+    double A1(0.08757), c1(-0.5257), t01(2.4013), s1(0.6721);
```

Comment:
Well, these were never documented to begin with... I just restored the HO pulse shape from earlier versions: https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_0_pre8/SimCalorimetry/HcalSimAlgos/src/HcalSiPMShape.cc

I can add a note like "HO SiPM pulse shape fit from Jake Anderson ca. 2013".